configure

```diff
@@ -551,5 +551,10 @@ eval VAL="\$${A}"
 [ -z "${VAL}" ] && VAL="(null)"
 echo " - ${A} = ${VAL}"
 done
+
+# This will not return correct values (tested):
```

Comment:
this change is doing nothing, and it is autogenerated, so it shouldnt be changed